src/App.tsx

```diff
@@ -1,25 +1,37 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
-import { getNumbers } from './utils';
-
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
-const items = getNumbers(1, 42).map(n => `Item ${n}`);
+import { Pagination } from './components/Pagination';
 
 export const App: React.FC = () => {
+  const [currentPage, onPageChange] = useState(1);
```

Comment:
The `useState` hook is incorrectly used here. `onPageChange` should be a function that updates `currentPage`, not a state variable itself. Consider using `const [currentPage, setCurrentPage] = useState(1);` and then passing `setCurrentPage` as `onPageChange` to the `Pagination` component.